addons/mail/models/mail_tracking_value.py

```diff
@@ -59,6 +59,11 @@ def create_tracking_values(self, initial_value, new_value, col_name, col_info):
                 'old_value_char': initial_value and initial_value.name_get()[0][1] or '',
                 'new_value_char': new_value and new_value.name_get()[0][1] or ''
             })
+        elif col_info['type'] in ['many2many', 'one2many'] :
+            values.update({
+                'old_value_char': initial_value and ',' .join(initial_value.mapped('name'))or '',
+                'new_value_char': new_value and ',' .join(new_value.mapped('name')) or ''
```

Comment:
Shouldn't it use `name_get()` instead? I'd also suggest adding a space after the comma, but that's an aesthetic option.